bundle/Resources/translations/ezplatform_fields_groups.fr_FR.xlf

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" datatype="plaintext" original="file.ext" target-language="fr">
+    <body>
```

Comment:
Hmm, what do you mean ? The comment might be missing a couple words :)

But I guess that you mean that the target language should be `fr_FR` ?  In which case you're right, I'll fix it.